starter-code/routes/drones.js

```diff
@@ -1,12 +1,28 @@
 const express = require('express');
-
+const Drone = require('../models/drone');
 // require the Drone model here
 
 const router = express.Router();
 
+// Iteration #2
 
 router.get('/drones', (req, res, next) => {
-  // Iteration #2
+  const droneNAme = req.params.droneNAme;
+
+  Drone.findByName(droneNAme, (err, drone) => {
```

Comment:
I think what you should have done here is:
```
Drone.find( droneName: droneNAme, (err, drone) => {
  if (err) { return next(err); }
  res.render('drones/show', { drone: drone })
});
```

I don't think `findByName` is a thing.